resources/page/permalinks_test.go

```diff
@@ -38,7 +38,7 @@ var testdataPermalinks = []struct {
 	{"/:filename/", true, "/test-page/"},                            // Filename
 	{"/:06-:1-:2-:Monday", true, "/12-4-6-Friday"},                  // Dates with Go formatting
 	{"/:2006_01_02_15_04_05.000", true, "/2012_04_06_03_01_59.000"}, // Complicated custom date format
-	// TODO(moorereason): need test scaffolding for this.
+	// TODO(more reason): need test scaffolding for this.
```

Comment:
Not this.  `moorereason` is my GH username.

Reply:
reverted